src/dymaptic.GeoBlazor.Core/Scripts/dotNetBuilder.ts

```diff
@@ -44,7 +44,7 @@ import {
     DotNetBookmark,
     DotNetViewpoint,
     DotNetField,
-    DotNetDomain, DotNetCodedValueDomain, DotNetCodedValue, DotNetInheritedDomain, DotNetRangeDomain
+    DotNetDomain, DotNetCodedValueDomain, DotNetCodedValue, DotNetInheritedDomain, DotNetRangeDomain, DotNetEffect
```

Comment:
should the formatting be consistest? either a single line per item or multiple items with a max length?

Reply:
You can tell my strong sense of formatting does not always transfer to TypeScript 🙂